### PR TITLE
release: v1.19.0

### PR DESCRIPTION
**Pull request by tejal29**
Release v1.19.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+**macOS**
+`curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/v1.19.0/skaffold-darwin-amd64 && chmod +x skaffold && sudo mv skaffold /usr/local/bin`
+
+**Windows**
+ https://storage.googleapis.com/skaffold/releases/v1.19.0/skaffold-windows-amd64.exe
+
+**Docker image**
+`gcr.io/k8s-skaffold/skaffold:v1.19.0`
+
+From release v1.19.0, skaffold will now collect anonymized Skaffold usage data.
```
Comment (MarlonGamez) [inline CHANGELOG.md:15]: We should mention that these are opt-in

**Review by MarlonGamez — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (IsaacPD) [reply CHANGELOG.md:15]: they are actually opt-out, not opt-in

Comment (MarlonGamez) [reply CHANGELOG.md]: whoops my bad, thanks for the correction

Comment (tejal29) [reply CHANGELOG.md]: done. added the summary. Want users to go to the doc link to know the command and example.

**Pull request merged**